src/components/ProcessForm/ProcessForm.tsx

```diff
@@ -221,6 +223,27 @@ export const ProcessForm: FunctionComponent<IProcessFormProps> = (props) => {
             </Col>
           </Form.Group>
         </fieldset>
+
+        <Form.Group as={Row} controlId="formOL">
+          <Col xl="6">
+            <Form.Label className="required">
+              <strong>OL</strong>
+            </Form.Label>
+          </Col>
+          <Col xl="6">
+            <Form.Control
+              as="select"
+              value={process.OL ?? "WPAFB"}
```

Comment:
All new ones default to WPAFB, should this be the case, or if they aren't an owner, should we setOL and use that as the default for them when creating?